drivers/console/Kconfig

```diff
@@ -189,6 +200,35 @@ config UART_PIPE_ON_DEV_NAME
 	  for pipe UART.
 endif
 
+config UART_MCUMGR
+	bool
+	prompt "Enable mcumgr UART driver"
+	select UART_INTERRUPT_DRIVEN
+	default n
+	help
+	  Enable the mcumgr UART driver. This driver allows the application to
```

Comment:
In a non-mcumgr application, this option is presented as an available console driver.

The language used here should be such that it is clear to any user what this is, and/or easy for the user to find out what this is.

Googling mcumgr does not yield any good documentation, so I think we need to be using different language here.

Reply:
Hmm... "mcumgr" is the name of the library and protocol, so I am not sure what other language could be used here.  I could add a brief description of what mcumgr is.  Does that sound reasonable?

Reply:
Yes.

Reply:
I have added some additional context to this setting's help.